<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarReglas</class>
 <widget class="QWidget" name="AgregarReglas">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar regla manualmente</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Puerto de origen</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Puerto de destino</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP destino</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Interfaz</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>String</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Protocolo</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Acción</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo de inicio</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IP origen</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo de finalización</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QTimeEdit" name="timeEdit_2">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>24</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>82</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" rowspan="2" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Habilitar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>24</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>[icmp/tcp/udp]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>158</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REGLA DE FILTRADO</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GUARDAR REGLA</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CANCELAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>timeEdit_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>abrirMostrarreglas()</slot>
  <slot>mostrarTabla()</slot>
 </slots>
</ui>
